SWLOR.Game.Server/Feature/GuiDefinition/ViewModel/HoloNetViewModel.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Threading.Tasks;
+using Discord;
+using Discord.Webhook;
+using SWLOR.Game.Server.Service;
+using SWLOR.Game.Server.Service.GuiService;
+
+namespace SWLOR.Game.Server.Feature.GuiDefinition.ViewModel
+{
+    public class HoloNetViewModel : GuiViewModelBase<HoloNetViewModel, GuiPayloadBase>
+    {
+        public string HoloNetText
+        {
+            get => Get<string>();
+            set => Set(value);
+        }
+
+        public const int MaxHoloNetTextLength = 800;
+        private const int Price = 2500;
+
+        protected override void Initialize(GuiPayloadBase initialPayload)
+        {
+            HoloNetText = string.Empty;
+            WatchOnClient(model => model.HoloNetText);
+        }
+
+        public Action OnClickSubmit() => () =>
+        {
+            if (string.IsNullOrWhiteSpace(HoloNetText))
+            {
+                return;
+            }
+
+            var message = HoloNetText;
+
+            if (message.Length > 600)
```

Comment:
Same thing here - remove the hard coded 600 and use MaxHoloNetTextLength

Reply:
Did the change